client/src/modules/break_even_reference/templates/action.cell.html

```diff
@@ -0,0 +1,30 @@
+<div class="ui-grid-cell-contents text-right" uib-dropdown dropdown-append-to-body  data-row="{{row.entity.label}}">
+
+  <a href uib-dropdown-toggle data-action="open-dropdown-menu">
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-row-menu="{{row.entity.label}}" class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>	
+    <li class="dropdown-header text-ellipsis"  style="max-width:250px;">
+      <a> {{row.entity.label}} </a>
```

Comment:
Since we cannot click this element, it doesn't make sense to wrap it in an `<a>` tag.  Let's remove that.

Reply:
Also, it looks like this removes the "text-ellipsis" effect:
![TextEllipsis](https://user-images.githubusercontent.com/896472/54202623-bad63e80-44d0-11e9-8213-e363bdde63c0.PNG)